src/components/LabeledRadioButton/LabeledRadioButton.jsx

```diff
@@ -0,0 +1,109 @@
+import _ from 'lodash';
+import classNames from 'classnames';
+import React from 'react';
+
+import { bindClassNames } from '../../util/style-helpers';
+
+import RadioButton from '../RadioButton/RadioButton';
+
+const boundClassNames = bindClassNames('LabeledRadioButton');
+const {
+	bool,
+	func,
+	node,
+	object,
+	string,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["controls", "toggles"]}
```

Comment:
Add `extend: "RadioButton"` as a property to this JSON and it will merge all RadioButton props with this.